lib/tasks/create_user.rake

```diff
@@ -0,0 +1,14 @@
+desc 'Create user into database'
+task :create_user, %i[name email password] => :environment do |_t, args|
+  user = User.new(
+    name: args[:name],
+    email: args[:email],
+    password: args[:password]
```

Comment:
Persist the `password` as the MD5-hashed representation of the provided plain text